### ISSUE TITLE
Understanding CFD simulation workflow in Butterfly

### ISSUE THREAD

**Issue by MRAOH** [mostaphaRoudsari/Butterfly#82]
Hi @mostaphaRoudsari  and @TheodoreGalanos ,
Thank you very much for developing these wonderful tools.
I am currently a beginner in CFD and would like to boost my understanding in workflow of using the Butterfly in building design.
I trying to use example files, but I've some misunderstanding about components.

Does have any resources to understanding the concepts, definitions and meanings?
Thank you very much in advance.

MRAOH

**Comment by mostaphaRoudsari** [comment]
Hi @MRAOH, Good timing for asking for resources. What about starting from [Learning resources](https://github.com/mostaphaRoudsari/Butterfly/wiki/Learning_Resources)?

Learning about Aerodynamics and fluid flows will be the basis for using CFD. Check the videos under [CFD Basics and CFD Intermidiate](https://github.com/mostaphaRoudsari/Butterfly/wiki/Learning_Resources#cfd-and-openfoam). 

I also just updated [Getting Started with Butterfly](https://github.com/mostaphaRoudsari/Butterfly/wiki/Getting-Started-with-butterfly) which hopefully is more informative than the previous version.

**Comment by MRAOH** [comment]
Hi @mostaphaRoudsari , Thank you very much for your extremely helpful and detailed response to my question! Really wonderful useful resources. 

I am planning for study related to CFD with thermal comfort in building design.
And I am eager to using Butterfly in my study.
I know that the "temperature" does not supported yet.
May I ask 1.when is possible to analysis with this capability in Butterfly?
and 2.Do you have any suggestion for me?
Thank you very much.

Best regards,
M

**Comment by TheodoreGalanos** [comment]
@MRAOH We have been planning to include temperature for a few weeks now. Unfortunately, I've delayed it terribly due to real life events that are currently taking up a lot of my time.

It is quite simple to add it so I think we can have it up and running this weekend if we manage to have a small development meetup.

I'll keep you updated here.

Kind regards,
Theodore.

**Comment by mostaphaRoudsari** [comment]
I'm going to close this issue in favor of #86.

We made a good progress on the development. Temperature field has been added and also the recipe for heatTransfer, but it can break some of the current workflows if we commit it to the repository. It will be added after we address #84.

![image](https://cloud.githubusercontent.com/assets/2915573/19226453/80844d86-8e79-11e6-91cd-db87898dcbab.png)
